enatega-multivendor-admin/src/components/Addon/Addon.jsx

```diff
@@ -124,7 +112,8 @@ function Addon(props) {
       )
       addons[index].quantityMinimumError =
         addons[index].quantityMinimumError ||
-        addons[index].quantityMinimum > addons[index].quantityMaximum
+        addons[index].quantityMinimum > addons[index].quantityMaximum ||
```

Comment:
The zero check is not linked with issue where minimum value showing error, although it is less than maximum. The error must be due to the fact that state is not updating. Please check and let me know if there is any confusion.